README.md

```diff
@@ -57,15 +57,15 @@ and run it:
 * git add
 * git push
 
-## Development environment
+## Staging environment
 
 If you are integrating feature branches into `staging`, run:
 
-    npm run deploy
+  `npm run deploy transformap-editor.apps.allmende.io` or `npm run deploy transformap-editor.apps.allmende.io <name_of_branch>`
```

Comment:
s/editor/data/

Reply:
@almereyda this is the editor's code repository, which is exposed on transformap-editor.apps.allmende.io. Why should it be deployed to transformap-data.apps.allmende.io?